src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/context_relevance/trec_dl_passage_relevance_judges.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.    old_criteria = """
</pre><p>this is current criteria right? can you rename variable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
renamed to current_criteria